src/components/toolbox/passphraseInput/passphraseInput.css

```diff
@@ -1,24 +1,84 @@
 @import '../../../app/variables.css';
 @import '../../../app/mixins.css';
 
+.wrapper {
+  margin: 0;
+  position: relative;
+}
+
 .inputs {
+  width: 100%;
+  display: flex;
   flex-wrap: wrap;
+  flex-direction: row;
+  border: 1px solid var(--dark-border-color);
+  border-radius: 3px;
+  background-color: var(--color-white);
+  padding: 0px 24px;
+  box-sizing: border-box;
+  outline: none;
+
+  &:focus-within {
+    border-color: var(--color-ultramarine-blue);
+  }
+
+  & > .inputContainer {
+    width: calc(100% / 6);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    flex-grow: 1;
+    box-sizing: border-box;
+    color: var(--color-ghost);
+    margin: 15px 0 20px;
+
+    &:focus-within {
+      color: var(--color-ultramarine-blue);
+    }
 
-  & > span {
-    margin: 0 0 12px 0;
+    & .inputNumber {
+      width: 20px;
+    }
+
+    & .inputNumberError:focus-within,
+    & .inputNumberError {
+      color: var(--color-burnt-sienna);
+    }
 
     & input {
-      text-align: center;
+      border: none;
+      text-align: left;
+      padding: 0 1px !important;
+      height: 20px !important;
+
+      &:focus {
+        color: var(--color-ultramarine-blue);
+      }
+
+      &::placeholder {
+        font-family: sans-serif;
+      }
     }
   }
+
+  & > .inputContainer:nth-child(-n+6) {
```

Comment:
I just recalled we also support 24-word passphrase:
![Screenshot 2019-10-21 at 08 38 21](https://user-images.githubusercontent.com/1254342/67181984-5a9c6900-f3de-11e9-8b52-68f6def8a0b8.png)

To fix spacing of those:
```suggestion
  & > .inputContainer:nth-last-child(n + 7) {
```